lib/ferver/file_list.rb

```diff
@@ -41,11 +42,24 @@ def file_by_id(id)
     def find_files
       @files = []
 
-      Dir.foreach(configured_file_path) do |file_name|
+      puts "Recursive: |#{Ferver.configuration.recursive}"
+      files = Dir.foreach(configured_file_path)   if Ferver.configuration.recursive == false
+      files = Dir.glob(File.join(configured_file_path , "**", "*"))  if Ferver.configuration.recursive == true
```

Comment:
Line is too long. [110/80]<br>Unnecessary spacing detected.<br>Space found before comma.